attempt to remove execPipe function

CrypticSwarm opened the pull request:
The exec Pipe function looks to do about the same as what child_process.exec does.
I thought that it should be removed and use child_process.exec instead.

Currently this is not ready to merge.  At the moment the graphs images don't render correctly.  They are about 5% smaller.  Any ideas why this isn't working?

CrypticSwarm committed:
attempt to remove execPipe function

austinyun commented:
Is it possible that the exec version is actually the correct behavior and execPipe is for some reason rendering them unnecessarily large?

CrypticSwarm commented:
No, highly doubt it.  This code doesn't work. I was mainly posting up for any suggestions why it wasn't working the way I thought it should. (The images don't render in the browser.)

CrypticSwarm committed:
dot files now render correctly

Had a problem with encoding.
Exec contrary to the docs is handing back a string not a buffer.

CrypticSwarm commented:
This is now working correctly.  It was a problem with the encoding of the streams.  Should be ready to pull.
